examples/practice-typescript/src/components/Input/index.tsx

```diff
@@ -0,0 +1,15 @@
+import './index.scss';
+
+interface InputProps {
+  type: string;
```

Comment:
`type` is a keyword, pls avoid to use as name.

Reply:
Fixed